src/printer.ts

```diff
@@ -691,7 +691,8 @@ export class PugPrinter {
     val = await format(val, { parser, ...options });
     val =
       this.quotes === '"'
-        ? val.replaceAll(/(?<!\\)"/g, '\\"')
+        ? // Escape double quotes, but only if they are not already escaped
+          val.replaceAll(/(?<!\\)((?:\\\\)*)"/g, '$1\\"')
         : val.replaceAll("'", "\\'");
```

Comment:
What is different about single quotes that they don't need the patch?

Reply:
They are by far to uncommon in that situation and if anybody encounters that, they should open a bug ticket / fix it via PR